docs/docs/getting-started.server.mdx

```diff
@@ -636,6 +636,41 @@ well.
 
 See [Using MDX with Next.js][next-mdx] for more details.
 
+#### Astro
+
+[Astro](https://astro.build/) has an [official MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/).
+Astro recommends using [the `astro add` CLI](https://docs.astro.build/en/guides/integrations-guide/#automatic-integration-setup)
+to install dependencies and update your config automatically:
+
+```sh
+# Using npm
+npx astro add mdx
+# Using yarn
+yarn astro add mdx
+# Using pnpm
+pnpx astro add mdx
+```
+
+This base setup allows you to import Markdown,
+[Astro components](https://docs.astro.build/en/core-concepts/astro-components/),
+and other MDX files as components.
+
+**To add support for [Preact](https://docs.astro.build/en/guides/integrations-guide/preact/), [Vue](https://docs.astro.build/en/guides/integrations-guide/vue/), [Svelte](https://docs.astro.build/en/guides/integrations-guide/svelte/) and more in your MDX,**
```

Comment:
```suggestion
**To add support for [other UI framework components](https://docs.astro.build/en/core-concepts/framework-components/) (e.g. Preact, Vue, Svelte and more) in your MDX,**
```
Just a suggestion for consideration: Our generic "UI Framework Components" page is a better general introduction for how all our components work, and will link to each (and every!) of the supported frameworks there.